packages/frontend/src/components/Explorer/ExploreTree/TableTree/Tree/types.ts

```diff
@@ -31,6 +31,13 @@ export type TreeProviderProps = {
     itemsMap: Record<string, NodeItem>;
     selectedItems: Set<string>;
     missingCustomMetrics?: AdditionalMetric[];
+    itemsAlerts?: {
```

Comment:
nice, we could refactor the missing custom dimensions/metrics and other warnings to this generic alert